metricbeat/module/mysql/status/data.go

```diff
@@ -72,6 +72,58 @@ var (
 			"select": c.Int("Com_select"),
 			"update": c.Int("Com_update"),
 		},
+		"queries":   c.Int("Queries"),
+		"questions": c.Int("Questions"),
+		"handler": s.Object{
+			"commit":        c.Int("Handler_commit"),
+			"delete":        c.Int("Handler_delete"),
+			"external_lock": c.Int("Handler_external_lock"),
+			"mrr_init":      c.Int("Handler_mrr_init"),
+			"prepare":       c.Int("Handler_prepare"),
+			"read": s.Object{
+				"first":    c.Int("Handler_read_first"),
+				"key":      c.Int("Handler_read_key"),
+				"last":     c.Int("Handler_read_last"),
+				"next":     c.Int("Handler_read_next"),
+				"prev":     c.Int("Handler_read_prev"),
+				"rnd":      c.Int("Handler_read_rnd"),
+				"rnd_next": c.Int("Handler_read_rnd_next"),
+			},
+			"rollback":           c.Int("Handler_rollback"),
+			"savepoint":          c.Int("Handler_savepoint"),
+			"savepoint_rollback": c.Int("Handler_savepoint_rollback"),
+			"update":             c.Int("Handler_update"),
+			"write":              c.Int("Handler_write"),
+		},
+		"innodb_buffer_pool": s.Object{
```

Comment:
@aidarbek sorry I didn't think before about this, but I think that we could put the innodb metrics under its own group, in case we add more innodb-related metrics in the future. What do you think?
```suggestion
		"innodb": s.Object{
		    "buffer_pool": s.Object{
```

Reply:
That makes sense to me, will change that

Reply:
Thanks!